deepchecks/tabular/dataset.py

```diff
@@ -711,6 +711,8 @@ def features_columns(self) -> pd.DataFrame:
         pd.DataFrame
         """
         self.assert_features()
+        if len(self.features) == 1:
```

Comment:
I feel like this could cause problems - I would expect this to always return a DataFrame. Need to test this out on a full_suite on several models IMO
Also, if we decide this is OK - change the typing of the function return.